doc/rbd/rbd-kubernetes.rst

```diff
@@ -188,16 +208,23 @@ pool created above, the following YAML file can be used after ensuring that the
         parameters:
            clusterID: b9127830-b0cc-4e34-aa47-9d1a2e9949a8
            pool: kubernetes
+           imageFeatures: layering
            csi.storage.k8s.io/provisioner-secret-name: csi-rbd-secret
            csi.storage.k8s.io/provisioner-secret-namespace: default
+           csi.storage.k8s.io/controller-expand-secret-name: csi-rbd-secret
+           csi.storage.k8s.io/controller-expand-secret-namespace: default
            csi.storage.k8s.io/node-stage-secret-name: csi-rbd-secret
            csi.storage.k8s.io/node-stage-secret-namespace: default
         reclaimPolicy: Delete
+        allowVolumeExpansion: true
         mountOptions:
            - discard
         EOF
         $ kubectl apply -f csi-rbd-sc.yaml
 
+Note that in Kubernetes v1.14 and v1.15 volume expansion feature was in alpha
```

Comment:
Actually we dont need this mention, considering the mentioned versions of Kube is really old or `< n-3 `

Reply:
Well, the doc covers v1.13+, so I thought it wouldn't hurt.